src/components/subtitlesettings/subtitlesettings.js

```diff
@@ -201,6 +204,10 @@ function embed(options, self) {
                 sliderVerticalPosition.classList.add('focusable');
                 sliderVerticalPosition.enableKeyboardDragging();
             }, 0);
+
+            // Replace color picker
+            options.element.querySelector('#inputTextColor').closest('div').classList.add('hide');
+            options.element.querySelector('#selectTextColor').closest('div').classList.remove('hide');
```

Comment:
no `closest` in webOS 1.2
```suggestion
            dom.parentWithTag(options.element.querySelector('#inputTextColor'), 'DIV').classList.add('hide');
            dom.parentWithTag(options.element.querySelector('#selectTextColor'), 'DIV').classList.remove('hide');
```

Reply:
Oh thank you! I tried to install the webOs 1.2 emulator with no luck so I using the webOS 6.0 simulator instead.